src/jit/codegenarmarch.cpp

```diff
@@ -1617,11 +1766,29 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
         }
         else
         {
-            regNumber argReg = curArgTabEntry->regNum;
             genConsumeReg(argNode);
```

Comment:
Now I understand that you wanted to define special `genConsumeReg` function for `PutArgSplit` such as `genConsumeAddress`, `genConsumeRegs`, or `genConsumePutStructArgStk`, right? I also it's good approach because `PutArgSplit` consume register(s), and it will be better to handle liveness in this special function if need.